From 7b655b36b349f6db9a7b658d0075c468cff83c2e Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Sat, 8 Dec 2018 13:14:06 +0100
Subject: [PATCH] Fix call to usage which requires an int argument
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Otherwise compilation fails:

main.c: In function ‘main’:
main.c:158:7: error: too few arguments to function ‘usage’
       usage();
       ^~~~~
main.c:73:14: note: declared here
 static void  usage(int status) _CUPS_NORETURN;
              ^~~~~
make[1]: *** [../Makedefs:270: main.o] Error 1
---
 scheduler/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scheduler/main.c b/scheduler/main.c
index ce7df89c0..a7df4525f 100644
--- a/scheduler/main.c
+++ b/scheduler/main.c
@@ -155,7 +155,7 @@ main(int  argc,				/* I - Number of command-line args */
   for (i = 1; i < argc; i ++)
   {
     if (!strcmp(argv[i], "--help"))
-      usage();
+      usage(1);
     else if (argv[i][0] == '-')
     {
       for (opt = argv[i] + 1; *opt != '\0'; opt ++)
-- 
2.20.0.rc2

